README.md

```diff
@@ -15,23 +15,24 @@ with [tests](https://github.com/JuliaGPU/CLFFT.jl/blob/master/test/runtests.jl)
 ```julia
 import OpenCL
 import CLFFT
+import FFTW
 
-const cl = OpenCL
+const cl = OpenCL.cl
 const clfft = CLFFT
 
 _, ctx, queue = cl.create_compute_context()
 
 N = 100
-X = ones(Complex64, N)
-bufX = cl.Buffer(Complex64, ctx, :copy, hostbuf=X)
+X = ones(ComplexF64, N)
+bufX = cl.Buffer(ComplexF64, ctx, :copy, hostbuf=X)
 
-p = clfft.Plan(Complex64, ctx, size(X))
+p = clfft.Plan(ComplexF64, ctx, size(X))
 clfft.set_layout!(p, :interleaved, :interleaved)
 clfft.set_result!(p, :inplace)
 clfft.bake!(p, queue)
 
 clfft.enqueue_transform(p, :forward, [queue], bufX, nothing)  
 result = cl.read(queue, bufX)
 
-@assert isapprox(norm(result - fft(X)), zero(Float32))
+@assert isapprox(FFTW.norm(result - FFTW.fft(X)), zero(Float32))
```

Comment:
So this norm is specific to FFTW or is it coming from LinearAlgebra?

Reply:
both `FFTW.norm` and `LinearAlgebra.norm` return the same value

Reply:
Try to use LinearAlgebra then to avoid confusion. You can simply add `using LinearAlgebra` and use `norm` without prefixes.